packages/components/pages/f-contact-preferences/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.5.2
+------------------------------
+*November 15, 2021*
+
+### Added
+- Added mapper.
```

Comment:
May be worth adding a tiny bit more detail here? `added contact preferences mapper & unit tests`? :)